src/Representation/Organization.php

```diff
@@ -23,5 +23,6 @@ public function __construct(
         protected ?OrganizationDomainCollection $domains = null,
         protected ?UserCollection $members = null,
         protected ?IdentityProviderCollection $identityProviders = null,
+        protected ?string $alias = null,
```

Comment:
```suggestion
        #[Since('26.0.0')]
        protected ?string $alias = null,
```

Reply:
Done!

Reply:
You still gotta import the attribute 😄 

```
use Fschmtt\Keycloak\Attribute\Since;
```

Please run `composer cs:fix` after, it fixes the order if you don't exactly know where to put it.

Reply:
Ouch. Embarrassing. It's fixed!